rapidast.py

```diff
@@ -94,8 +96,12 @@ def run_scanner(name, config, args, scan_exporter):
     # Part 1: create a instance based on configuration
     try:
         scanner = class_(config, name)
-    except OSError as excp:
-        logging.error(excp)
+    except OSError as e:
```

Comment:
Since the `RapidastScanner` has a `state` property, shouldn't it be set to `scanners.State.ERROR` when an error occurs?

Reply:
already existing code. Could I suggest you creating a separate PR for a better approach?

Reply:
I just realized that if an exception is raised at this point, `scanner` might not have been initialized yet, so there's no point in attempting to set its state